Projects/App/GraphQL/Octopus/MemberDeviceRegister.graphql

```diff
@@ -0,0 +1,3 @@
+mutation MemberDeviceRegister($input: MemberDeviceRegisterInput!) {
+  memberDeviceRegister(input: $input)
+}
```

Comment:
For the ones where we never pass anything else, like you would never pass ANDROID here, I'd just do
```suggestion
mutation MemberDeviceRegister($token: String!) {
  memberDeviceRegister(input: { token: $token, platform: IOS })
}
```
What do you think?

And then that'd be just
```swift
let deviceMutationInput = OctopusGraphQL.MemberDeviceRegisterInput(
  token: deviceTokenString
)
```
Further down